JitsiConference.js

```diff
@@ -425,16 +426,6 @@ JitsiConference.prototype.addTrack = function (track) {
         }
     }
 
-    track.ssrcHandler = function (conference, ssrcMap) {
-        if(ssrcMap[this.getMSID()]){
-            this._setSSRC(ssrcMap[this.getMSID()]);
-            conference.room.removeListener(XMPPEvents.SENDRECV_STREAMS_CHANGED,
-                this.ssrcHandler);
-        }
-    }.bind(track, this);
-    this.room.addListener(XMPPEvents.SENDRECV_STREAMS_CHANGED,
-        track.ssrcHandler);
-
     if(track.isAudioTrack() || (track.isVideoTrack() &&
```

Comment:
yeah this is a bit awkward i guess...we don't call this method anywhere anymore but it was kept to not break the API.  Basically it gets called here as well because you don't call both flow (addTrack and replaceStream) for the same track...it's one or the other.